app/components/request_form/request_form.js

```diff
@@ -15,11 +15,22 @@ export default class extends Controller {
     this.updateMembersCount();
   }
 
+  insertPlaceholderAtCursor() {
+    const placeholder = '{{VORNAME}}';
```

Comment:
I favored adding the placeholder to the translations file like we discussed @phoeinx ...
At the moment we still have these two instances in JS land where they are hardcoded. I was looking into how to do this in a rails app and there is a gem [i18n-js](https://github.com/fnando/i18n-js) that we can use, but honestly, I'm not sure it's worth adding the complexity and another gem for a theoretical use case.

I really like that we have set up the project to make it easy to internationalize cause I have worked on a project where we need to find all hardcoded strings and add them to translation files and it was a lot of work, and not the fun type. However, to my knowledge we haven't ever had a use case where the application itself should be in another language and in this particular case it's not as easy as just moving the string to the translation file.

What do you think though?

Reply:
Do you think we should add a `TODO:` to be able to find these easier in the future?

Reply:
Good point. Imo as we don't really have it as a TODO on the horizon I think it's fine without a `TODO:`. Less clutter currently and if we only have to search in `.js` files for `i18n`able text I think that's an acceptable search space.